### PR TITLE
Wandbot zd

### PR DESCRIPTION
**Pull request by ArtsiomWB**
Calls ZenPy api in order to fetch zendesk tickets from discourse, process them and feed them through wandbot API. Then get a response and post it privately in Zendesk under the ticket that had that question. 

Latest Changes:
Added my script, changed all of the print statements in to logger that you use in slack app. 

I have not added my app to the `run.sh` because I wanted to ask you what is the best way of going about it?
What does the .env look like in replit?

### CODE REVIEW

File: src/wandbot/apps/ZD/ZDWandbot.py [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+    async def main(self):
+        # test tickets
+        # self.create_new_ticket("How Do I start a run?")
+        # self.create_new_ticket("Is there a way to programatically list all projects for a given entity?")
+        while True:
+            await asyncio.sleep(120)
+
+            new_tickets = self.fetch_new_tickets()
+            logger.info(f"New unanswered Tickets: {new_tickets}")
```
Comment (parambharat) [inline src/wandbot/apps/ZD/ZDWandbot.py:104]: Maybe just log the number of unanswered tickets here. Logging a list of tickets with all their content can fill-up the logs making debugging harder.

File: src/wandbot/apps/ZD/ZDWandbot.py [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+from functools import partial
+from wandbot.api.client import APIClient
+from wandbot.api.schemas import APIQueryResponse
+from wandbot.utils import get_logger
+
+import pandas as pd
+from dotenv import load_dotenv
+
+load_dotenv()
```
Comment (parambharat) [inline src/wandbot/apps/ZD/ZDWandbot.py:19]: This should be removed. We will not use a .env file in the deployment but instead store secrets in and env. See comment below on using a pydantic settings class for getting the secrets from the env.

**Review by parambharat — CHANGES_REQUESTED** [review]
@ArtsiomWB. Thank you for the PR. This is a good start. I made some suggestions. Also, please merge the main branch into your branch to get the latest upstream changes.

**Review by Dezzj — COMMENTED** [review]
Veiwed

**Comment by ArtsiomWB** [comment]
@parambharat, made the suggested changes and pulled main! :D

**Review by Dezzj — COMMENTED** [review]
Veiwed

**Review by parambharat — APPROVED** [review]
LGTM

**Pull request merged**